contracts/contracts/Poll.sol

```diff
@@ -259,17 +258,17 @@ contract Poll is Params, Utilities, SnarkCommon, Ownable(msg.sender), EmptyBallo
 
     actualStateTreeDepth = depth;
 
-    emit MergeMaciStateAq(mergedStateRoot, numSignups);
+    emit MergeMaciState(mergedStateRoot, numSignups);
   }
 
   /// @inheritdoc IPoll
-  function mergeMessageAqSubRoots(uint256 _numSrQueueOps) public onlyOwner isAfterVotingDeadline {
+  function mergeMessageAqSubRoots(uint256 _numSrQueueOps) public isAfterVotingDeadline {
```

Comment:
Just an item for discussion: we need to think about access control. After coordinator work, I can start working on this.

Reply:
Yes totally a good discussion to have. My thoughts here are: it doesn't really matter who calls these functions, first of all they can cost quite a lot of gas, so technically if someone wants to call them, they are doing the coordinator/operator a favour. Second, these functions can only be called after the voting period ended, so no message would be able to be sent anymore either way, so it would not influence the voting period.